betty/json/linked_data.py

```diff
@@ -18,7 +20,8 @@
     from betty.ancestry.link import Link
 
 
-_SchemaTypeT = TypeVar("_SchemaTypeT", bound=Schema, covariant=True)
+_T = TypeVar("_T")
+_SchemaTypeT = TypeVar("_SchemaTypeT", bound=Schema, covariant=True, default=Schema)
 
 
 class LinkedDataDumpable(Generic[_SchemaTypeT]):
```

Comment:
Document that the output MUST NOT contain private data because it is  meant for publication